src/axolotl/utils/config/__init__.py

```diff
@@ -170,7 +170,10 @@ def normalize_config(cfg):
         cfg.processor_config = (
             cfg.processor_config or cfg.base_model_config or cfg.base_model
         )
-        model_config = model_config.text_config
+        if hasattr(model_config, "text_config"):
+            model_config = model_config.text_config
+        elif hasattr(model_config, "get_text_config"):
+            model_config = model_config.get_text_config()
```

Comment:
I think this is set twice. Here and in `load_model_config()->check_model_config()` above